harrison/memory-gui.js

```diff
@@ -0,0 +1,116 @@
+var MemoryGui = (function() {
```

Comment:
I like that you've made your cards interesting here. The only trouble is that they may be too interesting. For someone that doesn't know Russian, it's difficult to ascertain what is actually supposed to match to what here, and for testing purposes I found that I was just clicking randomly until I found something that actually matched. So, good cards, but remember that you're possibly making a game for people that don't know Russian!

Your cards are flipping, and then flipping back once two dissimilar cards are showing, which is great. There is the capability of flipping more than two cards at once, which is something you can fix by shortening the time of your hideSoon function, or by disabling clicking any time there are two cards showing.

At present your remove function doesn't seem to quite be working the way you want it to. It removes the background CSS, but retains the content of the card. Ultimately we want this function to remove the entirety of the cards content, leaving just a blank space (by removing it or rendering it invisible - and untouchable).

You're already aware of your reset button not working, but another issue with the button is that it is currently overlapping the game, so you can't help but press it while trying to get at some of the cards. I like that you've made it more interesting then just a tiny button to press, but you'll want to mess with your #reset CSS and adjust its position as necessary.

Beyond that, when I manually refresh the page the cards are being placed randomly, so that's working great.

Some minor tinkering you might do involves fixing the size of your cards. Right now they inflate or deflate based on the size of what's showing which gives the board a somewhat choppy, fluctuating quality. Minor issue, but something you can fix with a few adjustments.

Overall good work, and I hope you spend some time fixing some of these things. Let us know if you have questions!